make/targets.mk

```diff
@@ -1,5 +1,5 @@
 OFFICIAL_TARGETS  = ALIENFLIGHTF3 ALIENFLIGHTF4 ANYFCF7 BETAFLIGHTF3 BLUEJAYF4 FURYF4 REVO SIRINFPV SPARKY SPRACINGF3 SPRACINGF3EVO SPRACINGF3NEO SPRACINGF4EVO SPRACINGF7DUAL STM32F3DISCOVERY
-SKIP_TARGETS     := ALIENWHOOP MOTOLABF4 OMNINXT
+SKIP_TARGETS     := ALIENWHOOP MOTOLABF4 OMNINXT AG3X
```

Comment:
My 'make it target independent' senses are starting to tingle on `SKIP_TARGETS`. Wondering if it would be feasible / better to have targets declare targets to be excluded in their `target.mk`?

Reply:
Good thinking. Let me try that.